### PR TITLE
Add height and width attributes to canvas tag

### PR DESCRIPTION
**Pull request by Jir4**

### CODE REVIEW

File: Interpreter/Html/Canvas.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+namespace Hoa\Xyl\Interpreter\Html;
+
+/**
+ * Class \Hoa\Xyl\Interpreter\Html\Canvas.
+ *
+ * The <canvas></canvas> component.
```
Comment (Hywan) [inline Interpreter/Html/Canvas.php]:
```diff
- The <canvas></canvas> component.
+ The `<canvas />` component.
```

File: Interpreter/Html/Canvas.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+class Canvas extends GenericPhrasing
+{
+  /**
+   * Attributes description.
+   *
+   * @var array
+   */
+  protected static $_attributes        = [
+      'width'  => parent::ATTRIBUTE_TYPE_NORMAL,
+      'height' => parent::ATTRIBUTE_TYPE_NORMAL
```
Comment (Hywan) [inline Interpreter/Html/Canvas.php]: `height` before `width` (alphabetic order), thanks!

**Comment by Jir4** [comment]
https://github.com/hoaproject/Xyl/blob/master/Interpreter/Html/Video.php#L56-L57

*flies away*

**Comment by Hywan** [comment]
@Jir4 Please, update the other file ;-).

File: Interpreter/Html/Canvas.php [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+ * @license    New BSD License
+ */
+class Canvas extends GenericPhrasing
+{
+  /**
+   * Attributes description.
+   *
+   * @var array
+   */
+  protected static $_attributes        = [
```
Comment (Hywan) [inline Interpreter/Html/Canvas.php]: CS: Please, run `hoa devtools:cs`. 4 spaces instead of 2.

**Comment by Jir4** [comment]
@Hywan Say me if it's ok for you ;)

File: Interpreter/Html/Canvas.php [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+ *
+ * The `<canvas />` component.
+ *
+ * @copyright  Copyright © 2007-2015 Hoa community
+ * @license    New BSD License
+ */
+class Canvas extends GenericPhrasing
+{
+    /**
+    * Attributes description.
```
Comment (Hywan) [inline Interpreter/Html/Canvas.php]: One space is missing before all `*` :-).

**Comment by Hywan** [comment]
Merged (https://github.com/hoaproject/Xyl/commit/e25a733882d7c9cc299e4447db5d5ee62b256ed6 and https://github.com/hoaproject/Xyl/commit/4e0b2f8559c32cb5dd7d496ce6f57441f8a9dbe6).

Thanks!

**Pull request closed without merging**